index.bs

```diff
@@ -8120,6 +8184,162 @@ JavaScript Array values.
 </div>
 
 
+<h4 id="js-async-iterable">Async iterables — async iterable&lt;|T|&gt;</h4>
+
+In the JavaScript binding, IDL [=async iterable type|async iterable=] values are represented by
+a [=struct=] with the following [=struct/items=]:
+
+* <dfn for="JS async iterable">object</dfn>, a JavaScript value
+* <dfn for="JS async iterable">method</dfn>, a JavaScript value
+* <dfn for="JS async iterable">type</dfn>, either "<code>sync</code>" or "<code>async</code>"
+
+
+<div id="js-to-async-iterable" algorithm="convert a JavaScript value to async iterable">
+    A JavaScript value |V| is [=converted to an IDL value|converted=]
+    to an IDL <a lt="async iterable">async iterable&lt;<var ignore>T</var>&gt;</a> value as follows:
+
+    1.  If |V| [=is not an Object=], then
+        [=JavaScript/throw=] a <l spec=ecmascript>{{TypeError}}</l>.
+    1.  Let |method| be [=?=] <a abstract-op>GetMethod</a>(obj, {{%Symbol.asyncIterator%}}).
+    1.  If |method| is undefined:
```

Comment:
nit: For consistency with elsewhere (including within this PR), I guess we should use `<emu-val>undefined</emu-val>` here and below?

Reply:
```suggestion
    1.  If |method| is <emu-val>undefined</emu-val>:
```

Reply:
Note you're still missing the instance two lines below.